lib/screens/map_view.dart

```diff
@@ -85,18 +88,43 @@ class _MapViewScreenState extends State<MapViewScreen> {
           MarkerLayerOptions(
             markers: [
               Marker(
-                  width: 40.0,
-                  height: 40.0,
+                  width: 110.0,
+                  height: 110.0,
                   point: latlng.LatLng(locationData?.latitude ?? 51.5,
                       locationData?.longitude ?? -0.09),
-                  builder: (ctx) => const Beastie())
+                  builder: (ctx) => const Avatar()),
+              spawnBeastie(),
+              spawnBeastie(),
+              spawnBeastie()
             ],
           ),
         ],
       );
     }
   }
 
+  Marker spawnBeastie() {
+    var random = Random();
+    double longitudeRange = 0.00085;
+    double latitudeRange = 0.00088;
+    int randNum = random.nextInt(2);
+    int sign = 0;
+    if (randNum == 0) {
```

Comment:
You can simplify this by using a ternary:
`int sign = randNum == 0 ? -1 : 1;`

Reply:
Great tip! I will definitely change that